application/src/main/resources/application-default.yml

```diff
@@ -0,0 +1,12 @@
+spring:
+  profiles: default
```

Comment:
I don't think this is necessary since the applicable profile is already selected by means of loading this file ...